src/router.js

```diff
@@ -25,7 +25,7 @@ export class Router {
     return false;
   }
 
-  registerViewPort(viewPort, name) {
+  registerViewPort(viewPort:any, name?:string) {
```

Comment:
We'll eventually need an interface for view ports, but in the meantime `Object` would be more appropriate than `any`.